proto/lbm/token/v1/tx.proto

```diff
@@ -16,39 +16,22 @@ service Msg {
   // - EventSent
   // - transfer (deprecated, not typed)
   // Throws:
-  // - ErrInsufficientFunds:
+  // - ErrInvalidRequest:
   //   - the balance of `from` does not have enough tokens to spend.
   rpc Send(MsgSend) returns (MsgSendResponse);
```

Comment:
Why does not this function change to `Transfer`?

Reply:
We cannot change the name of the endpoint from `Send` to `Transfer`, because it would require another protobuf review of the clients.